tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/server/config/JMSConfigurationTest.java

```diff
@@ -97,4 +111,56 @@ public void testSetupJMSConfiguration() throws Exception {
 
       server.stop();
    }
+
+   @Test
+   public void testHandshakeTimeoutWithValueSet() throws Exception {
+      JMSServerManager jmsServer;
```

Comment:
The JMSServerManager is deprecated so it's probably best not to use it in new tests.  Check out the "embedded-simple" example for how to programmatically instantiate a broker with a configuration file.

Reply:
Or don't even use a configuration file and just programmatically configure the acceptor.

Reply:
Good point with JMSServerManager, thanks. 
The issue is about unability to set timeout value using conf file so using it is intentional. Setting this value programmatically is done [here](https://github.com/apache/activemq-artemis/pull/1534/files#diff-70ef2b6fc0b6f5e37ea6023acce65b8c). Or did you think anything else?

Reply:
The problematic use-case wasn't necessarily with a configuration file, but simply using a URL (which can be done programmatically).  I changed the test a bit before I merged it.  It's a bit simpler this way.

Reply:
Aha. Thank you :)